### PR TITLE
Concept - Slice Boundaries & RFC1918

### PR DESCRIPTION
**Pull request by sayantani11**
Solves a part of #5 
Explains concept of slice boundaries and RFC1918 network addresses.

### CODE REVIEW

**Comment by sayantani11** [comment]
cc @divya-mohan0209 
I know the explanations need a lot of improvement, so I'll be awaiting feedback. Also I was confused if there should be a heading over the concepts or they'll be a part of the overview section.

**Comment by divya-mohan0209** [comment]
Hey @sayantani11, good day! I actually intended the Glossary section to be separate and not a part of the overview section since it will be like a reference guide.

File: versioned_sidebars/version-0.2.0-sidebars.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "version2": [
     "overview",
+    "glossary",
```
Comment (veenamj-avesha) [inline versioned_sidebars/version-0.2.0-sidebars.json:4]: @sayantani11  - can you add glossary at the end of the sidebar, please? 

Glossary is usually added at the end of a book (similar to Index).

**Review by veenamj-avesha — COMMENTED** [review]
Left a comment

File: versioned_docs/version-0.2.0/glossary.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Glossary
+
+1. Slice Boundaries: The time required for processing depends upon the amount of data. When the amount of data
+to be replicated is massive, its efficient to seperate the data into slices and process it seperately.
+Slice boundaries are basically resources, with instructions that maintain the slices and also manages communication
+of the pods and services across clusters, edges, clouds, data centers etc.
+
+2. RFC1918 network addresses: The rapid growth of internet led to the exhaustion of the available ipV4 addresses, as a solution subnets
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.2.0/glossary.mdx:8]: Can you rephrase it as `The rapid growth of Internet has led to the exhaustion ....`?

Comment (veenamj-avesha) [inline versioned_docs/version-0.2.0/glossary.mdx:8]: Change `ipV4` to `IPv4` 

(the terminology is as per industry standards)

File: versioned_docs/version-0.2.0/glossary.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+## Glossary
+
+1. Slice Boundaries: The time required for processing depends upon the amount of data. When the amount of data
+to be replicated is massive, its efficient to seperate the data into slices and process it seperately.
+Slice boundaries are basically resources, with instructions that maintain the slices and also manages communication
+of the pods and services across clusters, edges, clouds, data centers etc.
+
+2. RFC1918 network addresses: The rapid growth of internet led to the exhaustion of the available ipV4 addresses, as a solution subnets 
+were introduced that an organisation could use internally. RFC1918 are requests for comment that defines a set of private
+ipv4 addresses that are not routeable on the internet. Any router connected to the internet will discard packets of data
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.2.0/glossary.mdx:10]: Change `ipV4` to `IPv4` 

(the terminology is as per industry standards)

File: versioned_docs/version-0.2.0/glossary.mdx [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+defined on the RFC1918.
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.2.0/glossary.mdx:11]: Either it should be `... in the RFC1918 record`. or `.., in RFC1918`. 

I have always heard of referring RFC as specifications but for RFC1918, I could only find it to be referred to as a record.

cc @pdorbala

File: versioned_docs/version-0.2.0/glossary.mdx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+## Glossary
+
+1. Slice Boundaries: The time required for processing depends upon the amount of data. When the amount of data
+to be replicated is massive, its efficient to seperate the data into slices and process it seperately.
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.2.0/glossary.mdx:4]: Typo 

Change `seperately` as `separately`

File: versioned_docs/version-0.2.0/glossary.mdx [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+Slice boundaries are basically resources, with instructions that maintain the slices and also manages communication
+of the pods and services across clusters, edges, clouds, data centers etc.
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.2.0/glossary.mdx:6]: Replace `data centers etc.` with `data centers, and so on.

**Comment by sayantani11** [comment]
Hi @divya-mohan0209 
Sorry was unavailable for a few days!
I will be doing the changes asap.

**Comment by veenamj-avesha** [comment]
We don't have a glossary and the latest version is v 1.2. 

We have revamped the entire sidebar and we have taken the two glossary terms and see if we can add them in our docs. The tracker is https://avesha.atlassian.net/browse/AM-12896. 

cc @narmidm
